templates/web/bexley/waste/_confirmation_after.html

```diff
@@ -0,0 +1,27 @@
+[% IF report.category == 'Report missed collection' %]
+
+  <h2>Bins reported as missed:</h2>
+  <p>
+    <ul>
+      [% titles = [ report.title ]; titles.import( report.get_extra_metadata('grouped_titles') ) %]
```

Comment:
I feel I shouldn't have to, but I was really struggling to throw the title and grouped_titles together (because TT didn't like that the latter is an arrayref though it will automatically deref them in other contexts, or something).